modules/connected-to-disconnected-prepare-mirror.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/connected-to-disconnected.adoc
+
+[id="connected-to-disconnected-prepare-mirror_{context}"]
+= Preparing the cluster for mirroring 
+
+Before disconnecting your cluster, you must mirror, or copy, the images in the registries from which your cluster pulls images while your cluster is still connected. In order to mirror the images, you must prepare your cluster by:
+
+* Adding the mirror registry certificates to the list of trusted CAs on your host.
+* Creating a `.dockerconfigjson` file that contains your image pull secret, which is from `cloud.openshift.com" token, which you use to mirror the {product-title} registries. 
+* Creating an `ImageContentSourcePolicy` (ICSP) object that associates the mirrored repository to the source repository for pods. The ICSP redirects image pulls from online registries to your offline mirror registry.
+
+
+.Procedure
+
+. Configuring credentials that allow images to be mirrored:
+
+.. Add the registry CA certificate, in the simple PEM or DER file formats, to the list of trusted CAs. For example:
++
+[source,terminal]
+----
+$ cp ~/certificate-trust-examples/Cert-trust-test-ca.pem /usr/share/pki/ca-trust-source/anchors/
+----
++
+[source,terminal]
+----
+$ update-ca-trust
+----
+
+.. Extract the `.dockerconfigjson` file from the global pull secret:
++
+[source,terminal]
+----
+$ oc extract secret/pull-secret -n openshift-config --confirm --to=. 
+----
++
+.Example output
+[source,terminal]
+----
+.dockerconfigjson
+----
+
+.. Edit the `.dockerconfigjson` file to add your mirror registry and authentication credentials and save it as a new file:
+// copied from olm-accessing-images-private-registries
++
+[source,terminal]
+----
+$ cat .dockerconfigjson | \
+    jq --compact-output '.auths["<registry>:<port>/<namespace>/"] |= . + {"auth":"<token>"}' \//<1>
+    > new_dockerconfigjson
+----
+<1> Replace `<registry>:<port>/<namespace>` with the mirror registry details and `<token>` with your authentication credentials.
+
+. Create an `ImageContentSourcePolicy` file: 
```

Comment:
So this section is just talking about how to prepare your cluster to mirror images right? Creating an ICSP is not needed here. ICSP only comes in when you need to start pulling images from the mirrored registry for your workloads, so it should only be in the `configuring the  mirror registry` section below.